src/containers/Map/index.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react'
+import styles from './styles.sass'
+import L from 'leaflet'
+
+export default class Map extends Component {
+  componentWillReceiveProps() {
+    // update map objects here
+  }
+
+  shouldComponentUpdate() {
+    return false
```

Comment:
is this stopgap required? I'd not tweak performance before locating an actual bottleneck